src/main/java/Duke/storage/Storage.java

```diff
@@ -0,0 +1,66 @@
+package Duke.storage;
+
+import Duke.tasks.*;
+
+import java.io.*;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents the file used to store the TaskList
+ */
+public class Storage {
+
+    /** Default file path used if the user does not provide the file name. */
+    static String path = Paths.get(".").toAbsolutePath().toString();
+    private static final String DEFAULT_FILEPATH = path.substring(0, path.length() - 1) + "src/main/";
+
+    private String filePath = "";
+
+    public Storage() {
+        this.filePath = DEFAULT_FILEPATH + "data/tasks.txt";
+    }
+
+    @SuppressWarnings("ResultOfMethodCallIgnored")
+    public Storage(String filePath) {
+        try {
+            this.filePath = DEFAULT_FILEPATH + filePath;
+            File file = new File(this.filePath);
+            file.createNewFile();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void store(TaskList toStore) {
```

Comment:
Maybe you could consider writing javadocs for these more complicated public facing methods, to give others a clearer picture of what they do?